src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -55,9 +58,11 @@ function DeploymentConfigForm({ respondOnSuccess, isUnSet }) {
     const [obj, json, yaml, error] = useJsonYaml(tempFormData, 4, 'yaml', true);
     const [chartConfigLoading, setChartConfigLoading] = useState(null)
     const [showConfirmation, toggleConfirmation] = useState(false)
+    const [showReadme, setReadme] = useState(false)
```

Comment:
instead of `setReadme` could be `setShowReadme`

Reply:
Done